templates/index.html

```diff
@@ -91,6 +91,15 @@ <h2 class="active item">Tell Me About Coffee</h2>
     </div>
 </div>
 
+  <footer class="ui inverted grey footer segment">
+    <div class="ui center aligned container">
+      <div class="ui horizontal inverted small divided link list">
+        <a class="item" href='mailto:billy@billyfung.com?Subject=tellmeabout.coffee'>Contact</a>
+        <div class="item">By <a href='billyfung.com'>Billy</a> and <a href='jinpark.net'>Jin</a></div>
```

Comment:
Have to make these absolute urls or theyll go to `tellmeabout.coffee.com/billyfung.com`